Dockerfile

```diff
@@ -39,7 +38,21 @@ RUN ARCH=$(uname -m) && \
       postgresql-server-devel \
       qpid-proton-c-devel \
       ruby-devel \
-      wget && \
+      wget \
+      # For seeding ansible runner / ansible-galaxy
+      ansible \
+      # For ansible-venv
+      gcc \
+      krb5-devel \
+      libcurl-devel \
```

Comment:
This (and a couple of others) is a duplicate from above, but I don't want someone to delete the above one for core and not realize it's still needed for ansible-venv.  It doesn't affect the built to list it twice.